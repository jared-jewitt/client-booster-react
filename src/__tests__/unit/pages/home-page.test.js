import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '@/pages/home-page';

describe('HomePage - unit', () => {

  it('renders correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });

});
