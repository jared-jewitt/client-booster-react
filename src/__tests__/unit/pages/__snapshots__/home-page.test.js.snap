// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage - unit renders correctly 1`] = `ShallowWrapper {}`;
